### PR TITLE
Members of Legislative Councils, mayors, and corporators

### PR DESCRIPTION
**Pull request by alexsdutton**
This adds some examples of memberships for these positions, and hooks the positions into the boundary index.

### CODE REVIEW

File: build-check.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Integrity checks failed:
+#<struct Commons::Integrity::Check::Base::Error category=:position_id_message, message="Q48730651 was found in boundaries/build/index.json but not in boundaries/build/position-data.json", filename=#<Pathname:boundaries/build/position-data.json>>
```
Comment (jacksonj04) [inline build-check.txt]: Is this something we should do something about?

File: boundaries/build/position-data-query-results.json [hunk 2]
```diff
@@ -4191,17 +4191,17 @@
         },
         "positionSuperclass": {
           "type": "uri",
-          "value": "http://www.wikidata.org/entity/Q5096847"
+          "value": "http://www.wikidata.org/entity/Q2285706"
```
Comment (jacksonj04) [inline boundaries/build/position-data-query-results.json]: These look to be related to query order changing - is this something we can slate to fix? Not needed for this PR.

File: build-check.txt (hunk 1 above)
Comment (alexsdutton) [reply build-check.txt]: Yes. Probably a good idea, just for consistency's sake. It's probably an incomplete-data-in-WD issue, which I'll look into.

File: boundaries/build/position-data-query-results.json (hunk 2 above)
Comment (alexsdutton) [reply boundaries/build/position-data-query-results.json]: Ticketed as https://github.com/everypolitician/commons-builder/issues/115.

File: build-check.txt (hunk 1 above)
Comment (alexsdutton) [reply build-check.txt]: Fixed :-)

Comment (jacksonj04) [reply build-check.txt]: :shipit:

**Comment by alexsdutton** [comment]
Build check squelched, so I reckon this is good to merge.

**Pull request merged**